tests/test_embeds.py

```diff
@@ -80,6 +80,48 @@ def test_len(embed: Embed) -> None:
     assert len(embed) == 69
 
 
+def test_eq() -> None:
+    embed_1, embed_2 = Embed(), Embed()
+    assert embed_1 == embed_2
+
+    # attribute=MISSING should not affect __eq__ (with attribute i mean any attribute of an Embed object)
+    embed_1.color = Color(123456)
+    assert not embed_1 == embed_2
+
+    embed_1.color = MISSING
+    assert embed_1 == embed_2
+
+    embed_1.add_field(name="This is a test field", value="69 test 69")
+    embed_2.add_field(name="This is a test field", value="69 test 69", inline=False)
+    assert not embed_1 == embed_2
+
+    embed_1.clear_fields(), embed_2.clear_fields()  # type: ignore
+
+    embed_1.title, embed_2.title = MISSING, MISSING
+    assert embed_1 == embed_2
+
+    # testing if two embeds with all MISSING attributes are equal
```

Comment:
`color`/`colour` is the only field that's meant to be set to `MISSING`, the others should always have some value or be `None`

Reply:
So is this test useless? Does the lib do some check to the embeds values provided from Users? If no, a user passing `MISSING` could break the `__eq__` method, we should at least raise some type of error

Reply:
nvm it'll not break the `__eq__` method but still i think we should raise an error (excluding color/colour)

Reply:
A user should never really manually set things to missing in the first place. It's mainly used internally to tell the difference between whether a user explicitly passed `None` or did not pass anything at all. The only usecase for a user to do this, ever, is to spoof this behaviour in e.g. a subclass that replaces internal behaviour.

Reply:
> A user should never really manually set things to missing in the first place. It's mainly used internally to tell the difference between whether a user explicitly passed `None` or did not pass anything at all. The only usecase for a user to do this, ever, is to spoof this behaviour in e.g. a subclass that replaces internal behaviour.

Ok, so do i need to change the test as well? In the end it is a test that can prove that not even setting values ​​as `MISSING` can break `__eq__`